praktikumsplaner-frontend/frontend/src/Constants.ts

```diff
@@ -6,3 +6,4 @@ const API_BACKEND_BASE = "/api/backend-service";
 export const NWK_BASE = `${API_BACKEND_BASE}/nachwuchskraft`;
 export const MELDEZEITRAUM_BASE = `${API_BACKEND_BASE}/meldezeitraum`;
 export const PRAKTIKUMSSTELLE_BASE = `${API_BACKEND_BASE}/praktikumsstellen`;
+export const APP_SECURITY = true;
```

Comment:
for development it could be useful to turn sec off. You can bind the state of that var to a profile.